GameDefinition/PLAYING.md

```diff
@@ -527,6 +539,87 @@ influence the speed of HP attrition and if it outweighs the benefits
 of exploration.
 
 
+Managing inventory and crafting
+-------------------------------
+
+All items belonging to your team are stored either in the character's personal 
+equipment outfit or in the shared inventory stash that is accessible by every 
+team member. 
+Items that are in the personal equipment outfit can be accessed by or 
+are beneficial for this particular character only. Stashed items can be 
+accessed by all the team members except those that are not calm enough 
+or injured somehow. But no one has a direct advantage from having the items 
```

Comment:
Did we agree on "or suffer a debilitating trauma" or something like that?

Reply:
Yeah... Pardon me.